src/pages/tokens/[tokenid].js

```diff
@@ -433,7 +443,9 @@ export default function Token(props) {
         <TagList>
           <TreeTag
             key="created-at"
-            TreeTagValue={new Date(token.created_at).toLocaleDateString()}
+            TreeTagValue=<time dateTime={new Date(token.created_at)}>
+              {new Date(token.created_at).toLocaleDateString()}
+            </time>
```

Comment:
This prop should be in `{}` also the `dateTime` prop here

```suggestion
            TreeTagValue={
              <time dateTime={token.created_at}>
                {new Date(token.created_at).toLocaleDateString()}
              </time>
            }
```